### PR TITLE
New version

### PR DESCRIPTION
**Pull request by shadyvb**

### CODE REVIEW

File: game.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                    <span id="playerTime">0</span>
+                </p>
+                <p id="countTime">
+                    <span id="timing">00:00</span>
+                </p>
+            </div>
+            <div class="setting" id="setting">
+                <span id="pause" title="Pause"><i class="fa fa-pause"></i></span>
+                <span id="resume" title="Resume"><i class="fa fa-play"></i> </span>
+                <span id="stop"><i class="fa fa-stop" title="Stop"></i></span>
```
Comment (shadyvb) [inline game.html]: Inconsistency with `title` attribute.

File: game.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            </div>
+        </div>
+        <!--    Result popup    -->
+        <div class="result-popup" id="result-popup">
```
Comment (shadyvb) [inline game.html]: Remove duplicate `class` attributes

File: index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        <link rel="stylesheet" href="css/flash-font.min.css">
+        <link rel="stylesheet" href="css/form.css">
+    </head>
+    <body>
+        <!--    Pages Content    -->
+        <div class="nav" id="nav">
+            <!--    Main Page   -->
+            <div class="container" id="container">
+                <h1>So, You Think You Are
+                    <span id="fast">Fast</span>
```
Comment (shadyvb) [inline index.html]: prefix IDs

File: index.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+                    ?
+                </h1>
+                <button type="button" id="yes">Yes</button>
```
Comment (shadyvb) [inline index.html]: ID should be unique, `yes` should be changed here.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+                </h2>
+                <span id="back">Get Back</span>
+            </div>
+        </div>
+        <!--    Registration Form   -->
+        <div class="user-info-form" id="formBody">
+            <form action="game.html" method="GET" name="Get Started">
+                <fieldset>
+                    <legend><i class="fa fa-user"></i> Register Now</legend>
+                    <label>Your Name :</label>
```
Comment (shadyvb) [inline index.html]: `<label for=userName>...</label>`

File: index.html [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+                    <input type="text" name="userName" id="userName" autofocus required>
+                    <label>Your Age :</label>
+                    <input type="text" name="userAge" id="userAge" required>
+                    <i class="fa fa-spinner fa-spin fa-fw" id="spinner"></i><input  type="submit" value="Go" class="submit" id="submit">
+                    <span id="closeForm">Cancel</span>
```
Comment (shadyvb) [inline index.html]:
```
element.addEventListener( 'click', function( ev )  {
  ev.preventDefault();

  return false;
} );
```

File: js/app.js [hunk 7]
```diff
@@ -0,0 +1,297 @@
...
+  function endInterval () {
+    window.clearInterval(countDown);
+  }
+
+
+  function secondPass() {
+
+    var minutes =Math.floor( levelSeconds / 60 ),
+        remSeconds = levelSeconds % 60;
+    elements.timming.innerHTML = "0" + minutes + ":" + remSeconds ;
```
Comment (shadyvb) [inline js/app.js]: `timming` > `timing`

File: js/app.js [hunk 8]
```diff
@@ -0,0 +1,297 @@
...
+    return clicks;
+  }
+
+  function gameEngine () {
+
+    elements.box.style.display = 'none';
+    elements.body.className = className;
+    var time;
+
+      time = Math.random() * dataTime * 1000;
```
Comment (shadyvb) [inline js/app.js]: no need for a new line to assign the value, add it to the `var` line

File: js/app.js [hunk 9]
```diff
@@ -0,0 +1,297 @@
...
+    setTimeoutTwo = setTimeout(function () {
+      gameEngine();
+
+    }, time);
+  }
+
+
+  function playAgain () {
+    clicks = 0;
+    switch (className) {
```
Comment (shadyvb) [inline js/app.js]: rename variable for readability, `currentLevel` would be good

File: js/app.js [hunk 10]
```diff
@@ -0,0 +1,297 @@
...
+
+      case 'easy':
+        levelSeconds = '30';
+        gameEngine();
+        startCount();
```
Comment (shadyvb) [inline js/app.js]: those two functions need to be written once after switch

File: js/app.js [hunk 11]
```diff
@@ -0,0 +1,297 @@
...
+    }
+    console.log(levelSeconds);
+  }
+
+  document.getElementById('stop').addEventListener('click', playAgain);
+  //box.addEventListener('click', startCount);
+  elements.box.addEventListener('click', countClicks);
+  elements.box.addEventListener('click', clickMeEngine);
+  document.getElementById('resume').addEventListener('click', startCount);
+  document.getElementById('resume').addEventListener('click', gameEngine);
```
Comment (shadyvb) [inline js/app.js]: make a single function for both functions

File: js/form.js [hunk 12]
```diff
@@ -0,0 +1,79 @@
...
+  	getBack: document.getElementById('get-back'),
+    userName: document.getElementById('userName'),
+    userAge: document.getElementById('userAge'),
+    submit: document.getElementById('submit'),
+    spinner: document.getElementById('spinner')
+  };
+
+  //  go to the "Go Home" page
+  myElement.no.addEventListener('click', function () {
+      'use strict';
```
Comment (shadyvb) [inline js/form.js]: use strict does not need to be added to every function

**Review by shadyvb — COMMENTED** [review]
Fixes

File: game.html (hunk 1 above)
Comment (msharafjr) [reply game.html]: It's working ya 3am.

File: game.html (hunk 2 above)
Comment (msharafjr) [reply game.html]: Done!

File: index.html (hunk 3 above)
Comment (msharafjr) [reply index.html]: Done!

File: index.html (hunk 4 above)
Comment (msharafjr) [reply index.html]: Done!

File: index.html (hunk 5 above)
Comment (msharafjr) [reply index.html]: Done!

File: index.html (hunk 6 above)
Comment (msharafjr) [reply index.html]: Done!

File: js/app.js (hunk 7 above)
Comment (msharafjr) [reply js/app.js]: Done!

File: js/app.js (hunk 8 above)
Comment (msharafjr) [reply js/app.js]: Done awl wa7da 3shan bt3asabak.

File: js/app.js (hunk 9 above)
Comment (msharafjr) [reply js/app.js]: Done!

File: js/app.js (hunk 10 above)
Comment (msharafjr) [reply js/app.js]: Done!

File: js/form.js (hunk 12 above)
Comment (msharafjr) [reply js/form.js]: Done!

File: js/app.js (hunk 11 above)
Comment (msharafjr) [reply js/app.js]: Done!

File: game.html [hunk 13]
```diff
@@ -14,55 +14,55 @@
...
                 <p id="countTime">
                     <span id="timing">00:00</span>
                 </p>
             </div>
-            <div class="setting" id="setting">
-                <span id="pause" title="Pause"><i class="fa fa-pause"></i></span>
-                <span id="resume" title="Resume"><i class="fa fa-play"></i> </span>
-                <span id="stop"><i class="fa fa-stop" title="Stop"></i></span>
+            <div id="game-setting">
+                <span id="pause-btn-choise" title="Pause"><i class="fa fa-pause"></i></span>
```
Comment (shadyvb) [inline game.html:50]: Choice not Choise

File: js/app.js [hunk 14]
```diff
@@ -198,45 +181,44 @@
...
 
-        var comments = [
-            "Congrats," + "\n" + "You are faster than a TURTLE!",
-            "Come On," + "\n" +"Is That All You Have?!",
-            "Do you even know who FLASH is?!",
-            "ooh" + "\n" + "I slept twice"
-        ];
-        alert(comments[Math.round(Math.random() * 3)]);
+      var comments = [
+        'Congrats,\n You are faster than a TURTLE!',
```
Comment (shadyvb) [inline js/app.js:212]: This won't work

**Review by shadyvb — COMMENTED** [review]
Fix this in a new PR

File: js/app.js (hunk 14 above)
Comment (shadyvb) [reply js/app.js:212]: Okay, it works, still, follow my instructions 😄

Comment (msharafjr) [reply js/app.js:212]: Following your instructions, well, and you 'do NOT betting on things you don't sure' :smile:

**Pull request merged**